client/src/app/shared/shared-moderation/user-ban-modal.component.ts

```diff
@@ -36,6 +37,12 @@ export class UserBanModalComponent extends FormReactive implements OnInit {
   openModal (user: User | User[]) {
     this.usersToBan = user
     this.openedModal = this.modalService.open(this.modal, { centered: true })
+
+    let isSingleUser = true
```

Comment:
```
const areMultipleUsers = Array.isArray(this.usersToBan) && this.usersToBan.length > 1
```

Reply:
Won't this cause an error if `usersToBan` is a `User` and not `User[]`?

Reply:
No because the clause will stop at `Array.isArray` if it's a `User`